File: documentation/user/en/query/requirements/examples/fetching/referenceContentAllWithAttributes.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityPrimaryKeyInSet(105703)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			ReferenceContentAllWithAttributes()
        		)
        	)
        )
	)
);

File: documentation/user/en/query/requirements/examples/fetching/referenceContentAllWithAttributes.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityPrimaryKeyInSet(105703)
				),
				require(
					entityFetch(
						attributeContent("code"),
						referenceContentAllWithAttributes()
					)
				)
			)
		);
	}
);